scripts/src/public/auth.ts

```diff
@@ -15,3 +16,20 @@ export async function authenticate(req: express.Request): Promise<db.AuthToken>
 
 	return token;
 }
+
+export const authenticateUser = async function(req: express.Request, res: express.Response, next: express.NextFunction) {
+	const token = await authenticate(req);
+	httpContext.set("userId", token.userId);
```

Comment:
maybe store these values in the context only AFTER you found the user?

Reply:
This way I can use the values as soon as possible so if for some reason the user doesn't exists I can debug it with these values